new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -168,13 +168,44 @@ public static boolean installTraefik(TraefikParams params) {
   }
 
   /**
-   * Create Treafik Ingress.
+   * Create Traefik Ingress
+   * @param params - the params to helm install command, including release name, chartDir and wls domain namespace
+   * @param domainUID - weblogic domainUID to create the ingress
+   * @param traefikHostname - the hostname for the ingress
```

Comment:
No need for dashes between parameter name and description in Javadoc.

Reply:
done